platform/qt/app/mapwindow.hpp

```diff
@@ -43,20 +44,20 @@ protected slots:
 
     QPointF m_lastPos;
 
-    QMapboxGLSettings m_settings;
-    std::unique_ptr<QMapboxGL> m_map{};
+    QMapLibreSettings m_settings;
+    std::unique_ptr<QMapLibreGL> m_map{};
 
-    QPropertyAnimation *m_bearingAnimation;
-    QPropertyAnimation *m_zoomAnimation;
+    QPropertyAnimation *m_bearingAnimation{};
+    QPropertyAnimation *m_zoomAnimation{};
 
-    unsigned m_animationTicks = 0;
-    unsigned m_frameDraws = 0;
+    unsigned m_animationTicks{};
+    unsigned m_frameDraws{};
```

Comment:
It's C++ initializer syntax. It initialised the variable to a default value (most commonly zero or `false`) but the type is always correct (otherwise one should really use `0u`, although this is probably optimised by the compiler).

Reply:
Thanks :)